bitcoin/src/pow.rs

```diff
@@ -18,12 +18,22 @@ use units::parse;
 use crate::blockdata::block::BlockHash;
 use crate::consensus::encode::{self, Decodable, Encodable};
 use crate::consensus::Params;
-use crate::error::{ContainsPrefixError, MissingPrefixError, PrefixedHexError, UnprefixedHexError};
+use crate::error::{ContainsPrefixError, MissingPrefixError, ParseIntError, PrefixedHexError, UnprefixedHexError};
 
 /// Implement traits and methods shared by `Target` and `Work`.
 macro_rules! do_impl {
     ($ty:ident) => {
         impl $ty {
+            /// Creates `Self` from a prefixed hex string.
```

Comment:
In f019e24f1f4c58f85177d5d77647dcbf23941b02:

I'm not totally thrilled with `Self` in a doccomment like this. Maybe if we put it in [] rustdoc will make it link to the right thing at least. But it's not a big deal, I'll ack anyway.